redis/src/test/scala/zio/redis/ApiSpec.scala

```diff
@@ -20,6 +24,15 @@ object ApiSpec
     with HashSpec
     with StreamsSpec {
 
+  case class RedisExecutorWrapper(re: RedisExecutor.Service)
```

Comment:
This is where I make the second redis executor by wrapping the service of the original

Reply:
```suggestion
  final case class RedisExecutorWrapper(re: RedisExecutor.Service)
```